pallets/dapp-staking-migration/src/lib.rs

```diff
@@ -373,17 +373,19 @@ pub mod pallet {
         pub(crate) fn migrate_ledger() -> Result<Weight, Weight> {
             match OldLedger::<T>::drain().next() {
                 Some((staker, old_account_ledger)) => {
-                    let locked = old_account_ledger.locked;
+                    let old_locked = old_account_ledger.locked;
 
                     // Old unbonding amount can just be released, to keep things simple.
                     // Alternative is to re-calculate this into unlocking chunks.
```

Comment:
Is the comment still relevant ? I guess we are not going for the _Alternative_

Reply:
Not relevant 🙂 

But we can delete the whole pallet after this is merged.